gcp-repo/src/main/resources/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
```

Comment:
Is there an Eclipse reason to use HTML 3.2? Google tends to recommend HTML 5 and personally I lean toward XHTML.